Classes/BEMCheckBox.h

```diff
@@ -8,6 +8,7 @@
 
 #import <UIKit/UIKit.h>
 
+@class BEMCheckBoxGroup;
```

Comment:
Is the only reason to use `@class` here instead of a regular `#import` statement to avoid a circular reference?
If so, either this `@class` declaration, or the one on BEMCheckBoxGroup should be able to be changed to `#import`.